plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/TestingExasolServer.java

```diff
@@ -0,0 +1,156 @@
+    {
+        container = startContainer();
+    }
+
+    private static ExasolContainer<?> startContainer()
+    {
+        ExasolContainer<?> container = new ExasolContainer<>().withRequiredServices(ExasolService.JDBC);
+        // Deactivate reuse to allow running test classes in parallel.
+        container.withReuse(false);
+        container.start();
+        return container;
+    }
+
+    public static TestingExasolServer start()
+    {
+        TestingExasolServer server = new TestingExasolServer();
+        server.createUserAndSchema();
+        return server;
+    }
+
+    private void createUserAndSchema()
+    {
+        executeAsSys(List.of(format("CREATE USER %s IDENTIFIED BY \"%s\"", TEST_USER, TEST_PASS),
+                format("GRANT CREATE SESSION TO %s", TEST_USER),
+                format("GRANT CREATE SCHEMA TO %s", TEST_USER),
+                format("GRANT CREATE TABLE TO %s", TEST_USER),
+                format("GRANT IMPORT TO %s", TEST_USER),
+                format("GRANT CREATE VIEW TO %s", TEST_USER),
+                format("GRANT CREATE ANY VIEW TO %s", TEST_USER)));
+        execute(List.of(format("CREATE SCHEMA %s", TEST_SCHEMA)), TEST_USER, TEST_PASS);
+    }
+
+    public String getJdbcUrl()
+    {
+        return container.getJdbcUrl();
+    }
+
+    public void execute(String sql)
+    {
+        execute(sql, TEST_USER, TEST_PASS);
+    }
+
+    private void executeAsSys(List<String> sql)
+    {
+        execute(sql, container.getUsername(), container.getPassword());
+    }
+
+    public void execute(String sql, String user, String password)
+    {
+        execute(List.of(sql), user, password);
+    }
+
+    public void execute(List<String> statements, String user, String password)
+    {
+        try (Connection connection = createConnection(user, password);
+                Statement statement = connection.createStatement()) {
+            for (String sql : statements) {
+                try {
+                    statement.execute(sql);
+                }
+                catch (SQLException e) {
+                    throw new RuntimeException("Failed to execute statement '" + sql + "'", e);
+                }
+            }
+        }
+        catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private Connection createConnection(String user, String password)
+            throws SQLException
+    {
+        return getConnectionFactory(getJdbcUrl(), user, password).openConnection(SESSION);
+    }
+
+    private ConnectionFactory getConnectionFactory(String connectionUrl, String username, String password)
+    {
+        DriverConnectionFactory connectionFactory = new DriverConnectionFactory(
+                new EXADriver(),
+                new BaseJdbcConfig().setConnectionUrl(connectionUrl),
+                StaticCredentialProvider.of(username, password));
+        return new RetryingConnectionFactory(connectionFactory);
+    }
```

Comment:
We basically use raw `DriverManager.getConnection` in testing server. You can refer to `TestingPostgreSqlServer`.